includes/classes/Indexable/User/User.php

```diff
@@ -639,18 +653,30 @@ public function query_db( $args ) {
 		 * way to do that.
 		 */
 		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared, WordPress.DB.DirectDatabaseQuery
-		$objects = $wpdb->get_results(
-			$wpdb->prepare(
-				"SELECT SQL_CALC_FOUND_ROWS ID FROM {$wpdb->users} {$orderby} LIMIT %d, %d",
-				(int) $args['offset'],
-				(int) $args['number']
-			)
+		$sql = $wpdb->prepare(
+			"SELECT SQL_CALC_FOUND_ROWS ID FROM {$wpdb->users} {$orderby} LIMIT %d, %d",
+			(int) $args['offset'],
+			(int) $args['number']
 		);
+		// phpcs:enable WordPress.DB.PreparedSQL.InterpolatedNotPrepared, WordPress.DB.DirectDatabaseQuery
+
+		/**
+		 * Filter user indexable DB query SQL.
+		 *
+		 * @hook ep_user_query_db_sql
+		 * @param {string} $sql The SQL query to be executed
+		 * @param {array} $args Query arguments
+		 * @since 2.5.0
+		 * @return {string} Modified SQL query
+		 */
+		$sql = apply_filters( 'ep_user_query_db_sql', $sql, $args );
+
+		// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared, WordPress.DB.DirectDatabaseQuery
+		$objects = $wpdb->get_results( $sql );
```

Comment:
```suggestion
		$objects = $wpdb->get_results( $sql ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared, WordPress.DB.DirectDatabaseQuery
```

If we phpcs:disable, we need to enable it again later.

Reply:
Good catch. I made the changes

Reply:
Did you push it though, @burhandodhy ?

Reply:
Nvm, I see you are enabling it again after the return